crates/pop-parachains/src/utils/git.rs

```diff
@@ -47,13 +49,13 @@ impl Git {
 	}
 	/// Clone `url` into `target` and degit it
 	pub fn clone_and_degit(
-		url: &str,
+		template: &Template,
```

Comment:
Not a fan of this. Git should not need to know about a `Template` type, only a url.

Reply:
You are right, should only pass a URL